docs/developers/how-to/building.rst

```diff
@@ -0,0 +1,81 @@
+==========================
+Build MicroOVN from source
+==========================
+
+Building from source
+--------------------
+
+MicroOVN is distributed as a snap and the only requirements for building it
+are ``Make`` and ``snapcraft``. You can install them with:
+
+.. code-block:: none
+
+   sudo apt install make
+   sudo snap install snapcraft --classic
+
+Snapcraft requires ``LXD`` to build snaps. So if your system does not have LXD
+installed and initiated, you can check out either `LXD getting started
+guides`_ or go with following default setup:
+
+.. code-block:: none
+
+   sudo snap install lxd
+   lxd init --auto
+
+To build MicroOVN, go into the repository's root directory and run:
+
+.. code-block:: none
+
+   make
+
+This will produce the ``microovn.snap`` file that can be then used to install
+MicroOVN on your system.
+
+Installing from local file
+--------------------------
+
+Following up on the previous section, you can install the ``microovn.snap``
+file with:
+
+.. code-block:: none
+
+   snap install --dangerous ./microovn.snap
+
+.. note::
+
+   If you are building latest MicroOVN from the ``main`` branch, it's possible
+   that it's using a non-stable core snap as its base. In that case, you may
+   get a message like this:
+
+   .. code-block:: none
+
+      Ensure prerequisites for "microovn" are available (cannot install snap base "core24": no snap revision available as specified)
+
+   In such a case, you will need to install the required core snap manually
+   from the ``edge`` risk level.. For example:
+
+   .. code-block:: none
+
+      snap install core24 --edge
+
+   Then repeat the installation step.
+
+You will also need to manually connect required plugs, as ``snapd`` won't
+do it automatically for locally installed snaps.
+
+.. code-block:: none
+
+   # Switch to root account
+   sudo -i
+   for plug in firewall-control \
+                hardware-observe \
+                hugepages-control \
+                network-control \
+                openvswitch-support \
+                process-control \
+                system-trace; do \
+       snap connect microovn:$plug;done
+   exit
+
```

Comment:
Can you have some kind of verification section that confirms to the user that everything went well?

Verify the install

Reply:
I'll add verification for correctly connected plugs. That should cover also correct installation.